prow/cmd/deck/main.go

```diff
@@ -172,6 +186,19 @@ func prodOnlyMain(configAgent *config.Agent, o options, mux *http.ServeMux) *htt
 	mux.Handle("/log", gziphandler.GzipHandler(handleLog(ja)))
 	mux.Handle("/rerun", gziphandler.GzipHandler(handleRerun(kc)))
 
+	if o.spyglass {
+		//TODO: Need to support authenticated buckets, #8910
+		c, err := storage.NewClient(context.Background(), option.WithoutAuthentication())
+		if err != nil {
+			logrus.WithError(err).Fatal("Error getting GCS client")
```

Comment:
Why do we just continue on without a GCS client?

Reply:
This is the way that the other two clients are handled in `prodOnlyMain`.